devbox.json

```diff
@@ -0,0 +1,20 @@
+{
+  "packages": [
+    "go_1_20@latest",
+    "kubectl@latest",
+    "tilt@latest",
+    "k3d@latest",
```

Comment:
Curious to know, what's the limit here? You list k3d, but why not docker or vscode? Genuinely asking!

Reply:
docker is a good addition, however not sure if we should package vscode, as not everyone uses the editor. I was trying get a minimal set of tools need for the project to work. Other developer tool like k9s, github-cli, glow, and fish or useful scripts that we like beyond make are also options.